docs/faq/faq.md

```diff
@@ -0,0 +1,106 @@
+# FAQ
+
+This section contains the answers to the most frequently asked questions by the community (Slack, GitHub, etc.).
+
+## Table of contents
+
+* [General](GeneralSection)
+  * [Cluster limits](ClusterLimits)
+  * [Why DaemonSets pods (e.g., Kube-Proxy, CNI pods) scheduled on Virtual Nodes are in OffloadingBackOff?](DaemonsetBackOff)
+* [Installation](InstallationSection)
+  * [Upgrade the Liqo version installed on a cluster](UpgradeLiqo)
+  * [How to install Liqo on DigitalOcean](InstallLiqoDO)
+* [Peering](PeeringSection)
+  * [How to force unpeer a cluster?](ForceUnpeer)
+  * [Is it possible to peer clusters using an ingress?](PeerOverIngress)
+
+(GeneralSection)=
```

Comment:
Pay attention because (if I remember correctly) those anchors are global and not file-scoped. If that, you should add a prefix (e.g. FAQGeneralSection)